klippy/extras/heaters.py

```diff
@@ -263,7 +263,7 @@ def lookup_heater(self, heater_name):
         return self.heaters[heater_name]
     def setup_sensor(self, config):
         modules = ["thermistor", "adc_temperature", "spi_temperature",
-                   "bme280", "htu21d", "lm75"]
+                   "bme280", "htu21d", "lm75", "raspberry_pi_temperature"]
```

Comment:
What do you think about shortening that to "rpi_temperature"?

Reply:
Yup